app/routes/podcasts/components/PodcastEditor.js

```diff
@@ -75,7 +75,11 @@ class PodcastEditor extends Component<Props, *> {
             filter={['users.user']}
             component={SelectInput.AutocompleteField}
           />
-          <Button disabled={pristine || submitting} submit>
+          <Button
+            success={!this.props.new}
```

Comment:
Perhaps extract this with the rest of the props?

Reply:
Sure, it was previously done this way because `new` is reserved